### PR TITLE
Set to baseline

### PR DESCRIPTION
**Pull request by benadam11**
Sets typography to an 8pt baseline grid and adds a debug component.

### CODE REVIEW

File: docs/assets/main.css [hunk 1]
```diff
@@ -114,8 +115,49 @@ a:visited {
...
+li,
+nav,
+ol,
+p,
+pre,
+table,
+tfoot,
+ul,
+video {
+  margin-top: 0;
```
Comment (raphamorim) [inline docs/assets/main.css]: If it's reseting it's better set `margin: 0;`. What do you think?

File: docs/assets/main.css [hunk 2]
```diff
@@ -1,8 +1,20 @@
 /* reset */
 * {
+  box-sizing: border-box;
```
Comment (raphamorim) [inline docs/assets/main.css]: shouldn't put `margin: 0; padding: 0` ?

File: packages/web/components/Primitives/GlobalReset.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  th,
+  thead,
+  time,
+  tr,
+  tt,
+  u,
+  ul,
+  var,
+  video {
+    margin: 0;
```
Comment (raphamorim) [inline packages/web/components/Primitives/GlobalReset.js]: 👏 🥇

File: docs/src/index.js [hunk 4]
```diff
@@ -21,6 +24,7 @@ function App() {
   const [baseline, setBaseline] = useState(true);
   return (
     <Fragment>
+      <GlobalReset />
```
Comment (raphamorim) [inline docs/src/index.js]: `CSSStylesReset` isn't better?

Comment (benadam11) [reply docs/src/index.js]: I think I renamed to <GlobalStyles />

**Review by raphamorim — APPROVED** [review]

**Pull request merged**